lightning/src/ln/features.rs

```diff
@@ -253,6 +261,8 @@ mod sealed {
 		"Feature flags for `payment_secret`.");
 	define_feature!(17, BasicMPP, [InitContext, NodeContext],
 		"Feature flags for `basic_mpp`.");
+	define_feature!(25, ShutdownAnySegwit, [InitContext, NodeContext],
```

Comment:
The spec PR has changed to 27.

Reply:
Thx for heads up, I'll rebase and readjust as the last step.